Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -130,7 +130,7 @@ def audit
       formula_count += 1
       problem_count += fa.problems.size
       problem_lines = format_problem_lines(fa.problems)
-      corrected_problem_count = options[:style_offenses]&.count(&:corrected?)
+      corrected_problem_count ||= options[:style_offenses]&.count(&:corrected?)
```

Comment:
Maybe this should always be set to an integer of some sort?

Reply:
https://github.com/Homebrew/brew/pull/4765/files#diff-bd0ba063e0f8aeb1f4a3997d34914c0cR77

It is being initialized as 0. Without this conditional assignment, its value is being changed to `nil`

Reply:
`0` isn't falsey so this means it'll never be set in this case. Instead use a different variable which can be the `count` or `nil` and use `.to_i` which will convert `nil` to `0.

Reply:
Fixed.